articles/modules/ROOT/pages/using-subqueries-to-avoid-the-eager.adoc

```diff
@@ -0,0 +1,222 @@
+= Using Subqueries to Scope and Avoid Eager Behavior
+
+:slug: using-subqueries-to-avoid-the-eager
+:author: Andrew Bowman
+:neo4j-versions: 5.x, 4.4, 4.3, 4.2, 4.1
+:tags: cypher, performance, load-csv
+:category: cypher
+
+Eager operators in a query plan can be disruptive, especially when performing writes involving large amounts of data, or batch loading.
+
+If you've used `USING PERIODIC COMMIT LOAD CSV` to import data into Neo4j, it's likely at some point that you've been bitten by the Eager:
+Some operations require eagerly pulling in interim results for all rows, which effectively disables the `PERIODIC COMMIT` behavior, possibly causing you to go out of memory when running on a large input dataset.
```

Comment:
explain why it's necessary (to separate the read and write horizon -> i.e. do all writes first or do all reads first)
that's why it has to pull all data from the top through the operation, because that effectively will create a "materialized" horizon

(you don't need to mention that neo4j doesn't support MVCC)